src/en/clean-copy/04-Part III. The New Hope/27.md

```diff
@@ -9,48 +9,48 @@ The first half of the 20th century, with all the wars, depressions, and prohibit
+And third, with his relentless advocacy and passion for the beer *alchemy*, Fritz Maytag encouraged many people to rebel against the world of identical pale lagers. California gradually became the center of what was later named ‘the craft revolution.' Many other beer enthusiasts started to open their microbreweries inspired by Maytag and the Anchor Brewing Company example[ref:{"source":{"alias":"acitelli-audacity","short":"Acitelli, T. (2013)","extra":["The Audacity of Hops: the History of America’s Craft Beer Revolution","ISBN 9781613743881"]}}:"‘The Last Shall Be First’":]().
 
-#### How to taste
+#### How to Taste
 
-Though Maytag sold the Anchor Brewing Company in 2010, his work lives on. The American steam beer aka the California common is being made by many small independent breweries (just like Fritz was dreaming of): American Toppling Goliath, Widmer Brothers, Smuttynose, Port City Brewing, and European Mikkeler and To Øl. Of course, ‘Anchor Steam’ should be considered the canonical one.
+Though Maytag sold the Anchor Brewing Company in 2010, his work lives on. The American Steam Beer aka the California Common is being made by many small independent breweries (just like Fritz was dreaming of): American Toppling Goliath, Widmer Brothers, Smuttynose, Port City Brewing, and European Mikkeler and To Øl. Of course, ‘Anchor Steam’ should be considered the canonical one.
 
-#### On ‘craft’ and ‘The Craft Revolution’
+#### On ‘Craft’ and ‘The Craft Revolution’
 
-In the previous chapters, we had used the phrase ‘craft breweries’ quite a handful of times, and this part of the book will be exclusively dedicated to them, so it would be nice to actually define ‘craft’ before telling its history.
+In the previous chapters, we have used the phrase ‘craft breweries’ quite a handful of times, and this part of the book will be exclusively dedicated to them, so it would be nice to actually define ‘craft’ before telling its history.
 
-Alas, it's not that simple. Early craft brewers were defining themselves through quality: we make better beer and do not use cheap ingredients. But as we all understand, it's not entirely true: a brewery is a commercial venture which has to economize; the question is just where the borderline is, e.g. which cost-saving practices are acceptable and which are not.
+Alas, it's not that simple. Early craft brewers defined themselves through quality and a *hands-on* approach to production. They  made better beer and did not use cheap ingredients. However, as we all understand, it's not entirely true. A brewery is a commercial venture which must economize. The question is just where the borderline is: e.g. which cost-saving practices are acceptable and which are not.
```

Comment:
I switched to hand-crafted. As a brewer in the 90's that was how we defined ourselves. Bigger companies, even New Belgium in 1998, were completely automated. 

If you prefer, it can be removed.